extensions/wikia/CommunityPage/templates/insightsModule.mustache

```diff
@@ -1,20 +1,30 @@
 <section class="community-page-insights-module" data-tracking="community-page-insights-{{type}}">
-	<h3 class="community-page-insights-module-header">{{title}}</h3>
-	{{#fulllistlink}}
-		<a class="community-page-insights-module-full-list-link" href="{{fulllistlink}}" data-tracking="view-full-list">
-			{{insightsModules.messages.fulllist}}&nbsp;&rarr;
-		</a>
-	{{/fulllistlink}}
+	<h3 class="community-page-insights-module-header">
+		<div class="community-page-insights-module-header-icon">
+			<svg width="48" height="48">
+				<use xlink:href="/extensions/wikia/CommunityPage/images/expand-article.svg#expand-icon"></use>
```

Comment:
Can you pass this path from the controller? 
I don't see a reason to include it in template directly since we know we'll be changing it depending on the card type